File: robocode2-bot-api-java/src/main/java/net/robocode2/events/WonRoundEvent.java
package net.robocode2.events;

import lombok.Builder;
import lombok.EqualsAndHashCode;
import lombok.ToString;
import lombok.Value;

/** Event occurring when a bot has won the round */
@Value
@EqualsAndHashCode(callSuper = true)
@ToString(callSuper = true)
public class WonRoundEvent extends Event {

  @Builder
  private WonRoundEvent(int turnNumber) {
    this.turnNumber = turnNumber;
  }
}

File: robocode-tankroyale-bot-api-csharp/events/WonRoundEvent.cs
using System.Collections.Generic;

namespace Robocode.TankRoyale.BotApi
{
  /// <summary>
  /// Event occurring when a bot has won the round.
  /// </summary>
  public sealed class WonRoundEvent : Event
  {
    /// <summary>
    /// Constructor.
    /// </summary>
    /// <param name="turnNumber">Turn number.</param>
    public WonRoundEvent(int turnNumber) : base(turnNumber)
    {
    }
  }
}